corehq/apps/linked_domain/tests/test_linked_apps.py

```diff
@@ -75,6 +75,17 @@ def setUp(self):
 
 
 class TestLinkedApps(BaseLinkedAppsTest):
+    def _make_master1_build(self, release=True):
```

Comment:
nit: it might be clearer to always specify `release`, since it's not obvious which would be the default behavior

Reply:
fair point, will update

Reply:
0ffe39bb0db8c00f2d42bfee17128e954ccb4aa2